cpp/src/centrality/betweenness_centrality.cuh

```diff
@@ -110,8 +111,6 @@ class BC {
     nullptr;                  // array<VT>(|V|) stores the shortest path counter for the latest SSSP
   double *deltas_ = nullptr;  // array<result_t>(|V|) stores the dependencies for the latest SSSP
 
-  // FIXME: This should be replaced using RAFT handle
-  int device_id_        = 0;
   int max_grid_dim_1D_  = 0;
   int max_block_dim_1D_ = 0;
   cudaStream_t stream_;
```

Comment:
So this stream_ is set to handle_.get_stream(), isn't it? Then, why store both handle_ and stream_? This can be confusing (need additional effort to figure out stream_ here is actually handle_.get_stream())

Reply:
Updated in e6c5010